site/src/components/WorkspaceStats/WorkspaceStats.tsx

```diff
@@ -6,11 +6,11 @@ import { Link as RouterLink } from "react-router-dom"
 import { Workspace } from "../../api/typesGenerated"
 import { CardRadius, MONOSPACE_FONT_FAMILY } from "../../theme/constants"
 import { combineClasses } from "../../util/combineClasses"
-import { getDisplayStatus } from "../../util/workspace"
+import { getDisplayBuildStatus } from "../../util/workspace"
 
 const Language = {
   templateLabel: "Template",
-  statusLabel: "Status",
+  statusLabel: "Last job status",
```

Comment:
Wouldn't this be the place where we do want the workspace status showing? I thought we wanted the workspace status at the top of the workspace page, and the job status in the timeline.

Reply:
Ahhhhhh ok, I got it. I will fix it.